src/screens/shared/auth/SignUp.js

```diff
@@ -0,0 +1,166 @@
+import React from 'react';
+
+import { Button } from '@material-ui/core';
+
+import Dropdown from '@components/Dropdown';
+import FieldInput from '@components/FieldInput';
+import '@styles/SignUp.css';
+import { validateField, updateUserFields } from '@lib/utils';
+
+class SignUp extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = {
+      firstName: '',
+      lastName: '',
+      email: '',
+      password: '',
+      userTypes: '',
+      groupGapContact: false,
+      errors: {}
+    };
+    this.handleChange = this.handleChange.bind(this);
```

Comment:
You can use arrow functions in a react class component to bind to `this` without calling bind. (eg. `handleChange = (name) => { ... }`)